leap_c/examples/pendulum_on_a_cart/mpc.py

```diff
@@ -389,33 +339,19 @@ def export_parametric_ocp(
     ######## Solver configuration ########
     ocp.solver_options.integrator_type = "DISCRETE"
     ocp.solver_options.nlp_solver_type = "SQP"
-    ocp.solver_options.hessian_approx = (
-        "GAUSS_NEWTON" if cost_type == "LINEAR_LS" else "EXACT"
-    )
+    ocp.solver_options.hessian_approx = "GAUSS_NEWTON"
```

Comment:
Do we only want to use GN Hessian now, even when the cost_type is EXTERNAL? Is this even allowed?

Reply:
But it is not allowed for our External cost formulation I mean? (That is still there but not usable from the outside btw?)